.changeset/good-pianos-heal.md

```diff
@@ -0,0 +1,7 @@
+---
+'@crowdstrike/glide-core': minor
+---
+
+Menu can no longer be opened by setting its `open` attribute via the "click" handler of an element outside Menu.
+If you need to open Menu via the "click" handler of an element outside Menu, you can call `click()` on Menu's target.
+This change was made to support opening sub-Menu targets programmatically via `click()`.
```

Comment:
It was nice to be able to support setting `open` this way. But supporting it precluded consumers being able to call `click()` on sub-Menu targets. And there's an alternative to setting `open` this way, while there's no alternative to not being able to call `click()` on a sub-Menu target. So, overall, it's a net improvement for consumers.